added homeWork 2

SergeyLysovAPP opened the pull request.

SergeyLysovAPP committed:
strange Xcode file-optimization

SergeyLysovAPP committed:
added homeWork 2

iMrOwl commented on homeWork/Sources/main.swift:
```diff
@@ -7,5 +7,114 @@
+
+func isEven(numForCheck: UInt) -> Bool {
+    
+    if numForCheck % 2 == 0 {
```
можно через одну строчку ? :

iMrOwl commented on homeWork/Sources/main.swift:
```diff
@@ -7,5 +7,114 @@
+
+func isDivBy3(numForCheck: UInt) -> Bool {
+    
+    if numForCheck % 3 == 0 {
```
и тут ?:

iMrOwl commented on homeWork/Sources/main.swift:
```diff
@@ -7,5 +7,114 @@
+//MARK: Задание 1
+//Написать функцию, которая определяет, четное число или нет.
+
+func isEven(numForCheck: UInt) -> Bool {
```
четные же могут быть отрицательными?)

SergeyLysovAPP committed:
Для функций isEven и isDivBy3:
- изменил тип входных данных с UInt на Int
- заменил полную форму оператора IF на тернарную
